src/main/java/com/musseukpeople/woorimap/post/application/PostService.java

```diff
@@ -25,4 +28,23 @@ public Long createPost(Couple couple, List<Tag> tags, CreatePostRequest createPo
         Post post = createPostRequest.toPost(couple, tags);
         return postRepository.save(post).getId();
     }
+
+    @Transactional
+    public Long modifyPost(List<Tag> list, EditPostRequest editPostRequest) {
```

Comment:
```suggestion
    public Long modifyPost(List<Tag> tags, EditPostRequest editPostRequest) {
```
list보단 tags가 더 좋은 네이밍인 것 같습니다.